packages/react/package.json

```diff
@@ -44,7 +44,6 @@
     "@types/react-dom": "*",
     "jest": "^29.3.0",
     "jest-environment-jsdom": "^29.3.1",
-    "react-test-renderer": "^18.2.0",
```

Comment:
Bit surprising that we no longer need it but it looks like the tests pass with no issues :) 
Is it that we're no longer using any snapshot tests in clerk/shared?

Reply:
yes, there are no snapshot tests in shared.